Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -489,6 +490,11 @@ public class EOSSessionsManager : IEOSSubManager
         Queue<Action> UIOnLeaveSession;
         Queue<Action> UIOnSessionSearchCompleted;
 
+        /// <summary>
+        /// Event to inform UI that it should refresh displayed Friends and Joining/Inviting buttons.
+        /// </summary>
+        public UnityEvent UIOnPresenceAffectingChange = new UnityEvent();
```

Comment:
I would name this simply `OnPresenceChange`. It is _used by_ the UI logic to make a change to the UI in _response to_ the presence state changing. It's possible that users or other areas of the code may want to subscribe to the event, and in that context this would be a confusion name.

Reply:
Hrmmrmmrmm.... This is another hard one. I can accept that, I guess.

What's the naming difference between "UnityEvent you can subscribe to (possibly for UI)" and "This is the function that runs On something happening (and is not an event to subscribe to) such as `OnRefreshSessionFindSessionsCompleteCallback`? I guess the Callback at the end?

Reply:
The difference (in my mind) should be in the _tense_ of the verb in concert with the callback. So `OnPresenceChange` is _subscribable_, and a naming convention that consumers might use for their function that is triggered on this event would be `OnPresenceChangedCallback`. One indicates that an action is happening, the other implies that it's happening to _resolve_ the action of the presence changing. Curious what @andrew-hirata-playeveryware would have to say about this - he's a fan of grammar and these kinds of discussions.

Also to be super-duper clear - this is a tiny point of discussion. Overall these changes are great. I just noticed the opportunity for a conversation. Having to think more explicitly around it reveals to me that there is a degree to which I can't articulate exactly my reasoning - which is an excellent sign that I should understand that instinct better, and change my mind if I can't articulate it.

Reply:
Ah, I didn't mean to imply that "this is frustrating to implement". Mostly just that I hadn't had protocols for the naming of these kinds of things before, and trying to reconcile it with past information. You're good, and talking about small details is what I was looking for in the PR.

Imma change this to OnPresenceChange, and make sure callback style event things end in -Callback.